packages/hardhat-ledger/src/internal/create-ledger-provider.ts

```diff
@@ -0,0 +1,17 @@
+import { EIP1193Provider, HardhatNetworkUserConfig } from "hardhat/types";
+
+import { LedgerProvider } from "../provider";
+import { withSpinners } from "./with-spinners";
+
+export function createLedgerProvider(
+  provider: EIP1193Provider,
+  networkConfig: HardhatNetworkUserConfig
+): LedgerProvider {
+  const accounts =
+    networkConfig.ledgerAccounts === undefined
+      ? []
+      : networkConfig.ledgerAccounts;
```

Comment:
This can be written as `networkConfig.ledgerAccoutns ?? []`. Maybe we should add the [`prefer-nullish-coalescing`](https://typescript-eslint.io/rules/prefer-nullish-coalescing/) rule (with `ignoreTernaryTests` set to false) to our eslint config. Do you want to do that? It will also mean changing a lot of code in other packages where we are not doing that, but the auto-fix of that rule seems to work well, so it's just a matter of running `yarn lint:fix` from the root once after adding the rule.

Reply:
Sure! only thing'd be that you'd have to review code from other packages here. I'll do it in a single commit so we can remove it if you feel like it messes up the review :)

Reply:
Yes, doing that in a single commit would be ideal.